README.md

```diff
@@ -430,7 +430,13 @@ skip:
   - nf_core_configs
 ```
 
-This will create a pipeline called `cool-pipe` in the directory `cool-pipes-company-cool-pipe` with `me` as the author. It will exclude all files required for GitHub hosting of the pipeline, the GitHub CI from the pipeline, remove GitHub badges from the `README.md` file, remove pipeline options related to iGenomes and exclude `nf_core/configs` options.
+This will create a pipeline called `coolpipe` in the directory `cool-pipes-company-coolpipe` (`<prefix>-<name>`) with `me` as the author. It will exclude all possible parts of the template:
```

Comment:
```suggestion
This will create a pipeline called `coolpipe` in the directory `myorg-coolpipe` (`<prefix>-<name>`) with `me` as the author. It will exclude all possible parts of the template:
```